src/mqtt.service.ts

```diff
@@ -215,7 +215,7 @@ export class MqttService {
       const f = fs.pop();
       const t = ts.pop();
       switch (f) {
-        // In case the filter level is '#', this is a match not matter whether
+        // In case the filter level is '#', this match always whether
```

Comment:
Or do you think this would be better "[…] this matches always whether the topic is undefined on this leve or not […]"